lib/screens/explore_screens/manage_invites/interactor/usecases/load_invites_use_case.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:async/async.dart';
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/api/invite_repository.dart';
+import 'package:seeds/datasource/remote/api/profile_repository.dart';
+import 'package:seeds/datasource/remote/model/invite_model.dart';
+
+class GetInvitesUseCase {
+  final InviteRepository _inviteRepository = InviteRepository();
+  final ProfileRepository _profileRepository = ProfileRepository();
+
+  Future<InvitesDto?> run() async {
+    final account = settingsStorage.accountName;
+
+    final result = await _inviteRepository.getInvites(account);
+
+    if (result.isError) {
+      return null;
+    } else {
+      final List<InviteModel> invites = result.asValue!.value;
+      final Iterable<Future<Result>> futures = invites
+          .where((element) => element.isClaimed && element.account != null)
+          .map((e) => _profileRepository.getProfile(e.account!));
+
+      final List<Result> accounts = await Future.wait(futures);
+
+      return InvitesDto(accounts, invites);
+    }
+  }
+}
+
+class InvitesDto {
+  final List<Result> accounts;
+  final List<InviteModel> invites;
+
+  InvitesDto(this.accounts, this.invites);
```

Comment:
const constructor

Reply:
can we enforce this with Lint?

Reply:
As far I know this lint is active so its weird